cmd/fleetctl/package_test.go

```diff
@@ -43,10 +45,44 @@ func TestPackage(t *testing.T) {
 	}
 
 	t.Run("deb", func(t *testing.T) {
```

Comment:
This is not just a test for `deb` now. We are testing this specific bug, maybe add a comment explaining what we are testing? (for future readers).